docs/100.yaml

```diff
@@ -224,7 +224,8 @@
   author : mathlib
 70:
   title  : The Perfect Number Theorem
-  decl   : nat.even_and_perfect_iff
+  links  :
+    mathlib archive : https://github.com/leanprover-community/mathlib/blob/master/archive/100-theorems-list/70_perfect_numbers.lean
```

Comment:
@bryangingechen what is the correct way of linking here?

Reply:
It works now, and at least one other Freek problem has this form, so I'll tell bors to merge.